docs/deploy-squid/migration.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 30
+title: Migration guide
+---
+
+# Migrate from older versions of `@subsquid/cli`.
+
+Since `@subsquid/cli@1.x` the commands `squid:deploy` and `squid:update` are deprecated in favor of the manifest-based deployments and `sqd deploy`.
+
+To migrate:
+
+- Create `squid.yaml` in the squid root folder
+- Copy the deployment manifest from e.g. [squid-evm-template](https://github.com/subsquid/squid-evm-template/blob/main/squid.yaml)
+- Update the squid name, description and version
```

Comment:
I just wanted to suggest to clarify that this point and the next 2 are to be performed on the `squid.yaml` file.
Something like:
 - copy the content of `squid.yaml` file from squid-evm-template
 - edit the local file, updating the squid name, description, and version
 - update (if necessary) ...

Reply:
maybe not just "copy file from", but also add a link to manifest docs?

Reply:
and why from `squid-evm-template`?

Reply:
i mean better to provide better to provide content of manifest right there: "copy the following lines"